bundle/Form/FieldTypeHandlerRegistry.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form;
+
+use OutOfBoundsException;
+use RuntimeException;
+use function gettype;
+use function is_callable;
+
+final class FieldTypeHandlerRegistry
+{
+    /**
+     * Map of registered callable or FieldTypeHandlerInterface objects.
+     */
+    private array $map;
+
+    /**
+     * Creates a service registry.
+     *
+     * In $map an array consisting of a mapping of FieldType identifiers to object / callable is expected.
+     * In case of callable factory FieldTypeHandlerInterface should be returned on execution.
+     */
+    public function __construct(array $map = [])
+    {
+        $this->map = $map;
+    }
+
+    /**
+     * Register a $service for FieldType $identifier.
+     *
+     * @param \Netgen\Bundle\InformationCollectionBundle\Form\FieldTypeHandlerInterface|callable $handler
+     */
+    public function register(string $identifier, $handler): void
+    {
+        $this->map[$identifier] = $handler;
+    }
+
+    /**
+     * Returns a FieldTypeHandlerInterface for FieldType $identifier.
+     *
+     * @throws \OutOfBoundsException
+     * @throws \RuntimeException When type is not a FieldTypeHandlerInterface instance nor a callable factory
+     */
+    public function get(string $identifier): FieldTypeHandlerInterface|\Netgen\Bundle\IbexaFormsBundle\Form\FieldTypeHandlerInterface
```

Comment:
Second return type hint was for Birthday field type? We should copy it's handler here once the tag names are updated, so the handlers are not in conflict.

Reply:
solved in e46037b4